ReactAndroid/src/androidTest/java/com/facebook/react/testing/ReactAppTestActivity.java

```diff
@@ -273,10 +264,6 @@ public void run() {
     }
   }
 
-  private ReactInstanceManager getReactInstanceManager() {
-    return mReactInstanceManager;
```

Comment:
@dulmandakh I'm not sure this is correct. We store a `mReactInstanceManager` in this class but with your change it will instead go to `ReactActivityDelegate` and call `((ReactApplication) getPlainActivity().getApplication()).getReactNativeHost()` (which is where it fails).

Is there a way we can retain the previous behavior here? It's also a bit odd to me to rely on different ways of getting the instance manager both within this class and through external usage when the getter is being called.

I'm not an expert here so I'd appreciate any help so I can better understand what is going on so I can make this work and land it.